policyengine_us/parameters/gov/states/mi/tax/income/credits/homestead_property_tax_credit/disabled_or_senior/not_refundable_rate.yaml

```diff
@@ -0,0 +1,41 @@
+description: Michigan allocated this fraction of the homestead property tax credit for disabled or senior filers as the non-refundable credit.
```

Comment:
This description is incorrect. It's a fraction of something else. Please correct it and say what it depends on.

Reply:
```suggestion
description: Michigan reduces the total amount of household resources by this fraction for disabled or senior filers, based on the income under the homestead property tax credit.
```

![Uploading image.png…]()